folding-schemes/src/lib.rs

```diff
@@ -122,6 +122,8 @@ pub enum Error {
     NoMultiInstances,
     #[error("Missing 'other' instances, since this is a multi-instances folding scheme. Expected number of instances, mu:{0}, nu:{1}")]
     MissingOtherInstances(usize, usize),
+    #[error("Type conversion error")]
+    TypeConversion,
```

Comment:
Maybe it's better to combine this with https://github.com/privacy-scaling-explorations/sonobe/blob/8a41e678223028edf9c89d4a1901df2eabfed54c/folding-schemes/src/lib.rs#L117-L118

Reply:
Oh right! Absolutely, updated: https://github.com/privacy-scaling-explorations/sonobe/pull/176/commits/22dd53df3b44bc9d1f725f9685e291c55de3183e

Reply:
Awesome! Now everything looks good to me :)